frontend/www/js/omegaup/arena/events_socket.test.ts

```diff
@@ -88,50 +85,31 @@ describe('socket', () => {
       await server.closed;
       expect(client.readyState).toBe(WebSocket.CLOSED);
 
-      const mySocket = new socket.EventsSocket(
+      const socket = new EventsSocket(
         Object.assign({}, options, { disableSockets: false }),
```

Comment:
```suggestion
        {...options, disableSockets: false},
```

igual en los demás constructores.

Reply:
Listo